web/content/docs/benchmarks/reactive-transport/kineticreactant_allascomponents/KineticReactant2.pandoc

```diff
@@ -0,0 +1,92 @@
+Respective material properties, initial and boundary conditions are listed in the tables below.
+
+**2d scenario:**
+The 2d--scenario only differs in the domain geometry and assignement of the boundary conditions.
+The horizontal domain is 0.5 m in x and 0.5 m in y direction, and,  discretized into 10374 quadratic elemtents with an edge size of 0.0025 m.
+
+![Domains for the 1d/2d simulation models](../KineticReactant2_domain.png)
+
+-----------------------------------------
+
+|Parameter | Description | Value | Unit |
+|:-------- | :---------- | :---- | :--- |
+| $\phi$   | Porosity    | 1.0   |  |
+| $\kappa$ | Permeability | 1.157e-12 | m^2^ |
+| $S$   | Storage | 0.0     |  |
+| $a_L$ | long. Dispersion length | 0.0   | m |
+| $a_T$ | transv. Dispersion length | 0.0  | m |
+| $\rho_w$ | Fluid density | 1.0e+3 | kg m^-3^ |
+| $\mu_w$ | Fluid viscosity | 1.0e-3 | Pa s |
+| $D_{\text{H}^+}$ | Diffusion coef. for $\text{H}^+$ | 1.0e-7     | m^2^ s |
+| $D_{solutes}$ | Diffusion coef. for Synthetica, Syntheticb and Productd | 1.0e-12 | m^2^ s |
+| $U$ | Reaction speed constant | 1.0e-3 | h^-1^ |
+| $K_m$ | Half--saturation constant | 10 | mol kg^-1^~water~ |
+
+Table: Media, material and component properties
+
+
+-----------------------------------------
+
+| Parameter | Description | Value | Unit |
+|:--------- | :---------- | :---- | :--- |
+| $p(t=0)$  | Initial pressure | 1.0e+5  | Pa |
+| $g_{N,downstream}^p$ | Water outflow mass flux | -1.685e-02 | mol kg^-1^~water~ |
+| $g_{D,upstream}^p$ | Pressure at inlet | 1.0e+5 | Pa |
+| $c_{Synthetica}(t=0)$  | Initial concentration of Synthetica | 0.5     | mol kg^-1^~water~ |
+| $c_{Syntheticb}(t=0)$  | Initial concentration of Syntheticb | 0.5     | mol kg^-1^~water~ |
+| $c_{Productd}(t=0)$  | Initial concentration of Productd | 0     | mol kg^-1^~water~ |
+| $c_{\text{H}^+}(t=0)$  | Initial concentration of $\text{H}^+$ | 1.0e-7     |  mol kg^-1^~water~ |
+|  $g_{D,upstream}^{Synthetica_c}$ | Concentration of Synthetica |  0.5 | mol kg^-1^~water~ |
+|  $g_{D,upstream}^{Syntheticb_c}$ | Concentration of Syntheticb |  0.5 | mol kg^-1^~water~ |
+|  $g_{D,upstream}^{Productd}$ | Concentration of Productd |  0.0 | mol kg^-1^~water~ |
+|  $g_{D,upstream}^{\text{H}^+}$ | Concentration of $\text{H}^+$ |  1.0e-7 | mol kg^-1^~water~ |
+
+Table: Initial and boundary conditions for the 1d scenario
+
+
+
+## Results
+
+The kinetic reaction results in the expected decline of the concentration of Synthetica and Syntheticb, which is super-positioned by the influx of these two educts through the left side.
+By contrast, the concentration of Productd increases in the domain.
+Over time, opposed concentration fronts for educts and Productd evolve.
+Both, OGS-6 and OGS-5 simulation yield the same results in the 1d as well as 2d scenario, which verify the implementation of OGS-6--IPhreeqc.
```

Comment:
It would be helpful, if you have some error bounds for the difference between the OGS-5 and OGS-6 solutions.

Reply:
I added error bounds for concentration of Productd in the 1d scenario.